app/src/main/res/layout/activity_hello_world.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context="com.ustwo.boilerplate.feature.helloworld.HelloWorldActivity">
+
+    <TextView
+            android:id="@+id/textView_helloWorld"
```

Comment:
Also probably out of scope here but this makes me think it would be good to standardise our id naming.